CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for `erlang:apply/2`
 - Support for `lists:keystore/4`
 - Support for `erlang:size/1` bif
+- Support for `maps:from_keys/2` NIF
```

Comment:
We probably could also mention the memory leak fix with OP_PUT_MAP_ASSOC.

Reply:
Added.